calblueprint/ogsc
# Add TextListItem profile field value type, update Disciplinary Actions to be a TextListItem

⚠️ **Some of these changes address PR feedback on #234.** This should be merged first into the upstream branch!

* Add TextListItem profile field value type, which includes only a required `comment` field and a `date` field
* Updates Disciplinary Actions to be a TextListItem-type field
* Reorganizes some components to remove some redundancy (PFEM now has `shouldToastOnSuccess` option out of the box)
* Updates edit icon to correct SVG

## How to Test/Use PR feature

* Re-seed database for the new structure of disciplinary actions
* Visit a player's profile and validate that the new Disciplinary Actions field is working as expected
* Visit the Create Player form and validate that Disciplinary Actions can be created, updated, and submitted with the new player

## PR Checklist

Does your branch (check if true):

## Related PRs

* #234: Missing "Add Internal Assessments" button now appears inline with the TestResultTable, in this PR. Also, the value history summary has been removed from the TestResultTable.

## Screenshots

![image](https://user-images.githubusercontent.com/2977329/115837776-ceea0b00-a3cd-11eb-89fb-e792d0af03ad.png)


CC: @sydbui
